cmd/fdsn-quake-consumer/main.go

```diff
@@ -129,6 +129,7 @@ func (n *notification) Process(msg []byte) error {
 		if err := e.save(); err != nil {
 			return fmt.Errorf("error saving SC3ML %s %s: %w", v.S3.Bucket.Name, v.S3.Object.Key, err)
 		}
+		log.Println("saved", e.PublicID)
```

Comment:
suggestion: a space would make the logs easier to read, i.e. `saved<space>`?

Reply:
`Println` separates different parameters with space already.
See https://github.com/golang/go/blob/master/src/fmt/print.go#L1216

Reply:
I see, thanks!